File: app/src/main/java/cc/smartcash/wallet/Models/TransactionDetails.java
package cc.smartcash.wallet.Models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.google.gson.annotations.SerializedName;

@JsonIgnoreProperties(ignoreUnknown = true)
public class TransactionDetails {
    @SerializedName("fromAddress")
    private String fromAddress;
    @SerializedName("toAddress")
    private String toAddress;
    @SerializedName("Amount")
    private Double Amount;

    public String getFromAddress() {
        return fromAddress;
    }

    public void setFromAddress(String fromAddress) {
        this.fromAddress = fromAddress;
    }

    public String getToAddress() {
        return toAddress;
    }

    public void setToAddress(String toAddress) {
        this.toAddress = toAddress;
    }

    public Double getAmount() {
        return Amount;
    }

    public void setAmount(Double amount) {
        Amount = amount;
    }
}

File: app/src/main/java/cc/smartcash/wallet/Models/TransactionDetails.kt
package cc.smartcash.wallet.Models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.google.gson.annotations.SerializedName
import java.io.Serializable

@JsonIgnoreProperties(ignoreUnknown = true)
class TransactionDetails : Serializable {

    @SerializedName("fromAddress")
    var fromAddress: String? = null

    @SerializedName("toAddress")
    var toAddress: String? = null

    @SerializedName("Amount")
    var amount: Double? = null

}
